fastq_filtrator.py

```diff
@@ -0,0 +1,113 @@
+def read_reads(input_fastq: str) -> list:
+    with open(input_fastq) as fastq_file:
+        reads_fast = fastq_file.readlines()
+
+        all_reads = []
+        seq = 0
+        for read in reads_fast:
+            seq += 1
+            if seq % 2 == 0 and seq % 4 != 0:
+                all_reads.append(read)
+        fastq_file.close()
+
+        return all_reads
+
+
+def gc_read(reads: list, borders=(0, 100)) -> list:
+    reads_count = []
+    for seq in reads:
+        gc_count = 0
+        for nucl in seq:
+            if nucl == 'G' or nucl == 'C':
+                gc_count += 1
+        gc_content = (gc_count / len(seq)) * 100

```

Comment:
👍не самый изящный, но зато самый эффективный способ. Кст, можно немного упростить
```suggestion
        gc_count = 0
        for nucl in seq:
            gc_count += (nucl == 'G' or nucl == 'C')
        gc_content = (gc_count / len(seq)) * 100

```